app/gui/src/project-view/components/DocumentationEditor.vue

```diff
@@ -75,6 +75,20 @@ const handler = documentationEditorBindings.handler({
     <div class="DocumentationEditor">
       <div ref="toolbarElement" class="toolbar">
         <FullscreenButton v-model="fullscreen" />
+        <SvgButton name="header1" title="Header 1" @click.stop="markdownEditor?.toggleHeader(1)" />
+        <SvgButton name="header2" title="Header 2" @click.stop="markdownEditor?.toggleHeader(2)" />
+        <SvgButton name="header3" title="Header 3" @click.stop="markdownEditor?.toggleHeader(3)" />
+        <SvgButton name="quote" title="Quote" @click.stop="markdownEditor?.toggleQuote()" />
+        <SvgButton
+          name="bullet-list"
+          title="Bullet list"
+          @click.stop="markdownEditor?.toggleList('unordered')"
+        />
+        <SvgButton
+          name="numbered-list"
+          title="Numbered list"
+          @click.stop="markdownEditor?.toggleList('ordered')"
+        />
```

Comment:
The block type should probably be a dropdown, top-level toggle buttons for everything would get unwieldy. We should be able to use the same menu code we used for Lexical:
https://github.com/enso-org/enso/pull/11469/files#diff-c9082ea1842a4e8a415bc841239cb0866d6d8ef678c957fe94b033e8cc8ef6d9